README.md

```diff
@@ -74,14 +76,21 @@ print(r.json())
 # {'cookies': {'foo': 'bar'}}
 ```
 
-Supported impersonate versions, as supported by [curl-impersonate](https://github.com/lwthiker/curl-impersonate):
+Supported impersonate versions, as supported by my [fork](https://github.com/yifeikong/curl-impersonate) of [curl-impersonate](https://github.com/lwthiker/curl-impersonate):
+
+However, only Chrome-like browsers are supported.
```

Comment:
you could be more explicit here to why not supported others.

Reply:
I agree, it would be great to have some documentation explaining why we don't support firefox or other ones supported by the curl_impersonate

Reply:
Possibly simply adding a link to https://github.com/yifeikong/curl_cffi/issues/59#issuecomment-1810410273 will be enough

Reply:
Appologies for the missunderstanding, I'll make sure it's well documented in the new version.

Reply:
Please never apologize! Thank you for the work on this mate its an amazing repository!